features/fixtures/maze_runner/Assets/Scripts/Main.cs

```diff
@@ -34,23 +34,37 @@ public class Main : MonoBehaviour
     [DllImport("NativeCrashy")]
     private static extern void crashy_signal_runner(float num);
 
+#endif
+
+#if UNITY_SWITCH
+
+    [DllImport("__Internal")]
+    private static extern int bugsnag_getArgsCount();
+
+    [DllImport("__Internal")]
+    private static extern string bugsnag_getArg(int index);
+
+
 #endif
 
     private const string API_KEY = "a35a2a72bd230ac0aa0f52715bbdc6aa";
     private Dictionary<string, string> _webGlArguments;
 
     private string _fakeTrace = "Main.CUSTOM () (at Assets/Scripts/Main.cs:123)\nMain.CUSTOM () (at Assets/Scripts/Main.cs:123)";
     private string _mazeHost;
+    private SwitchCacheType _switchCacheType = SwitchCacheType.R;
+    private int _switchCacheIndex = 0;
+    private string _switchCacheName = string.Empty;
 
     public void Start()
     {
         Debug.Log("Maze Runner app started");
-
+#if UNITY_SWITCH
+    GetSwitchArguments();
+#endif
```

Comment:
Would have been cleaner leaving it as the `#elif` on L52 (as was) and just changing L53 imo.

Reply:
Moved back